README.md

```diff
@@ -137,16 +137,16 @@ The `.py` source files are written in [Jupytext `percent` format](https://jupyte
 ## <a name="tutorials-index"> Tutorials Index </a>
 Here we provide an index pointing to different available tutorials by their task type, techniques, and integrations.
 * Task
-    * Text Classification (Text): `spam`, `weather`
+    * Text Classification (Text): `spam`, `crowdsourcing`
     * Relation Extraction (Text): `spouse`
-    * Visual Relationship Detection (Image): `visual`
+    * Visual Relationship Detection (Image): `scene_graph`
 * Techniques
-    * Labeling with Labeling Functions (LFs): `spam`, `spouse`, `visual`, `weather`
+    * Labeling with Labeling Functions (LFs): `spam`, `spouse`, `scene_graph`, `crowdsourcing`
     * Augmentation with Transformation Functions (TFs): `spam`
     * Monitoring with Slicing Functions (SFs): `spam`
-    * Using Crowdworker Labels: `weather`
-    * Multi-Task Learning (MTL): `mtl`, `visual`, `spam`
+    * Using Crowd Worker Labels: `crowdsourcing`
+    * Multi-Task Learning (MTL): `mtl`, `scene_graph`, `spam`
 * Integrations
-    * TensorFlow/Keras: `spam`
+    * TensorFlow/Keras: `spam`, `crowdsourcing`
```

Comment:
+spouse for keras

Reply:
Crowdsourcing goes in sklearn (we use pytorch-transformers to get BERT features, and then use sklearn).